src/main/java/BadPracticesModel.java

```diff
@@ -0,0 +1,68 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Using multiple Lombok annotations can lead to redundancy. The @Data annotation already includes @Getter, @Setter, @ToString, and @EqualsAndHashCode. (Severity: medium)
**Suggestion**: Remove @Setter, @Getter, and @ToString as they are redundant with @Data.